packages/docsify-server-renderer/README.md

```diff
@@ -35,12 +35,12 @@ renderer.renderToString(url)
   <meta charset="UTF-8">
   <title>docsify</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
-  <link rel="stylesheet" href="//unpkg.com/docsify/themes/buble.css" title="buble" disabled>
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify/themes/vue.css">
```

Comment:
Why removing the `disabled` and `title` ?

Reply:
`disabled` means the css is not loaded, right?

Reply:
yes, 
`buble` is a theme just like the `vue`. So we cant have both as it will not look good. So we need to disable one.

Reply:
But in fact, only `buble` is loaded here, and it's `disabled`🤣